timesheets/src/Code.js

```diff
@@ -20,112 +20,176 @@ var COLUMN_NUMBER = {
       + 1, numRows, 1);
-  dropdownValues = ['NOTIFIED', 'NOT NOTIFIED'];
+  dropdownValues = ['NOTIFIED', 'PENDING'];
   rule = SpreadsheetApp.newDataValidation().requireValueInList(dropdownValues)
       .build();
   notifiedColumnRange.setDataValidation(rule);
-  notifiedColumnRange.setValue('NOT NOTIFIED');
+  notifiedColumnRange.setValue('PENDING');  
+}
+
+/**
+ * Sends a notification email to employees, with the
+ * appropriate approval/disapproval subject and message.
+ *
+ * @param {String} email Employee's email.
+ * @param {String} emailSubject Appropriate email subject line regarding
+ * whether or not employee's timesheet has been approved.
+ * @param {String} emailMessage Appropriate email message body regarding
+ * whether or not employee's timesheet has been approved.
+ */
+function notify(email, emailSubject, emailMessage) {
+    MailApp.sendEmail(email, emailSubject, emailMessage);
 }
 
-/** Runs the script "Send Emails". Sends e mails to every employee notifying
- * them whether or not their timesheet has been approved.  Checks + updates
- * "NOTIFIED" status so employers do not send e mails more than once.
-*/
-function sendEmails() {
+/**
+ * Sets the notification status to NOTIFIED for employees
+ * who have received a notification email.
+ *
+ * @param {Object} sheet Current Spreadsheet.
+ * @param {Object} notifiedValues Array of notified values.
+ * @param {Integer} i Current status in the for loop.
+ * @parma {Integer} beginningRow Row where iterations began.
+ */
+function updateNotifiedStatus(sheet, notifiedValues, i, beginningRow) {
+  // Update notification status.
+  notifiedValues[i][0] = 'NOTIFIED';
+  sheet.getRange(i + beginningRow, COLUMN_NUMBER.NOTIFY).setValue('NOTIFIED');
+}
+
+/** 
+ * Checks the approval status of every employee, and calls helper functions
+ * to notify employees via email & update their notification status.
+ */
+function checkApprovedStatusToNotify() {
   var sheet = SpreadsheetApp.getActiveSheet();
   var lastRow = sheet.getLastRow();
   var lastCol = sheet.getLastColumn();
-  // lastCol here is the NOTIFIED column
+  // lastCol here is the NOTIFIED column.
   var frozenRows = sheet.getFrozenRows();
   var beginningRow = frozenRows + 1;
   var numRows = lastRow - frozenRows;
 
-  // gets ranges of values all at once
+  // Gets ranges of necessary values all at once.
   var emailValues = sheet.getRange(beginningRow, COLUMN_NUMBER.EMAIL, numRows, 1).getValues();
   var approvalValues = sheet.getRange(beginningRow, COLUMN_NUMBER.APPROVAL,
       lastRow - frozenRows, 1).getValues();
-  var notifiedValues = sheet.getRange(beginningRow, 12, numRows,
+  var notifiedValues = sheet.getRange(beginningRow, COLUMN_NUMBER.NOTIFY, numRows,
       1).getValues();
 
-  // goes through the whole email column
-  for (var i = 0; i <= lastRow - beginningRow; i++) {
-    // don't notify twice
+  // Traverses through email column.
```

Comment:
is this only going through the email column? looks like you check other columns as well